app/views/admin/auctions/index.html.erb

```diff
@@ -61,7 +61,7 @@
                                 </td>
                                 <td><%= I18n.l(auction.starts_at) %></td>
                                 <td><%= I18n.l(auction.ends_at) %></td>
-                                <td class="auction-highest-offer"><%= auction.highest_price %></td>
+                                <td class="auction-highest-offer"><%= auction.highest_offer_id.present? ? auction.highest_price : t('nan')%></td>
```

Comment:
- How about `t('.')` shortcut to avoid cluttering global namespace?
- Is there some more meaningful and abstract name for this? Something that would reveal the intention, like `no_price`?